packages/rc-mfa-config/src/components/home/index.tsx

```diff
@@ -98,10 +101,23 @@ export const HomePage: FC = () => {
     },
   })
 
+  useEffect(() => {
```

Comment:
For what its worth, I spoke to Will about this this morning (on my dev portal PR) and he pointed me towards rendering <Loader /> if token was null over useEffect. I don't know why, just wanted to flag that in case there's any specific reason to do it one way over another

Reply:
I don't get it? Surely the token will never be null? We'll redirect away to /login because home is within a private route